src/main/resources/db/migration/changes/Courses.json

```
@@ -0,0 +1,63 @@
+{
+    "databaseChangeLog": [
+      {
+        "changeSet": {
+          "id": "CoursesTable-1",
+          "author": "TomShangguan",
+          "preConditions": [
+            {
+              "onFail": "MARK_RAN"
+            },
+            {
+              "not": [
+                {
+                  "tableExists": {
+                    "tableName": "COURSES"
+                  }
+                }
+              ]
+            }
+          ],
+          "changes": [
+            {
+              "createTable": {
+                "columns": [
+                  {
+                    "column": {
+                      "autoIncrement": true,
+                      "constraints": {
+                        "primaryKey": true,
+                        "primaryKeyName": "COURSES_PK"
+                      },
+                      "name": "ID",
+                      "type": "BIGINT"
+                    }
+                  },
+                  {
+                    "column": {
+                      "name": "COURSEMAJOR",
+                      "type": "VARCHAR(255)"
+                    }
+                  },
+                  {
+                    "column": {
+                      "name": "COURSENUMBER",
```

```suggestion
                      "name": "COURSE_NUMBER",
```

Since you used camelCase in the original name, this has to be SNAKE_CASE.  That's just how liquibase (the database migration system) works.